### PR TITLE
new support lifecycle doc

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+![Support Lifecycle](./media/support-lifecycle/support-lifecycle.png)
+
+The following table provides examples of how the release cadence correlates with the support
+lifecycle:
+
+> [!NOTE]
+> The dates in the following table are for example purposes only. The actual release dates vary.
+
+| Version number | Release schedule | Breaking changes | Release type | Support type | Supported until version |
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:107]: @dcaro , I replaced my table with Mikes; however, this dropped the patch versions I had illustrated. I know we wanted azCLI and azPS to use the same release cadense, but is this chart correct for azCLI?

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+## Breaking changes
+
+Major version releases of Azure CLI introduce breaking changes and occur up to two times per
+year. Breaking changes may occur at any point for preview extensions. Non-GA extensions aren't
+required to adhere to breaking change policies.
+
+For detailed information about breaking changes in Azure CLI, see [Release notes](./release-notes-azure-cli.md).
+
+## Supported environments
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:137]: @mikefrobbins , This sections feels redundant to #operating-systems below

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+To learn more about Azure CLI extensions, see [Use and manage Azure CLI extensions](/cli/azure/azure-cli-extensions-overview).
+
+## Versioning
+
+Azure CLI follows [Semantic Versioning](https://semver.org/) for version numbering. Semantic versioning is a system that uses a three-part number format: `major.minor.patch`.
+
+### Core
+
+* **Major versions:** Breaking changes, including architectural and fundamental design changes. We provide critical security fixes to the last minor version of the previous major version for three years.
```
Comment (dcaro) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:39]:
```suggestion
* **Major versions:** Major changes, including architectural and fundamental design changes. We provide critical security fixes to the last minor version of the previous major version for three years.  
```

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+To learn more about Azure CLI extensions, see [Use and manage Azure CLI extensions](/cli/azure/azure-cli-extensions-overview).
+
+## Versioning
+
+Azure CLI follows [Semantic Versioning](https://semver.org/) for version numbering. Semantic versioning is a system that uses a three-part number format: `major.minor.patch`.
+
+### Core
+
+* **Major versions:** Breaking changes, including architectural and fundamental design changes. We provide critical security fixes to the last minor version of the previous major version for three years.  
+* **Minor versions:** Backward compatible feature updates and improved Azure service coverage.
```
Comment (dcaro) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:40]:
```suggestion
* **Minor versions:** Updated for feature updates and improved Azure service coverage.
```

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+Azure CLI **extensions** don't have a specific release cadence and need to be upgraded manually. 
+
+Breaking changes might not affect existing scripts, but it's a good practice to verify your scripts in a test environment before performing an upgrade. 
+
+> [!NOTE]
+> The release cadence might be slightly adjusted to be aligned with major announcements relevant to Azure.
+
+## Standard Term Support releases (STS)
+
+The STS versions of Azure CLI core give you the most recent capabilities. STS releases don't experience breaking changes for six months. Breaking changes are released in the second and fourth quarters, and usually align with Microsoft Build and Microsoft Ignite.
```
Comment (dcaro) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:64]:
```suggestion
The STS versions of Azure CLI core give you the most recent capabilities. STS releases don't experience breaking changes for six months. Breaking changes are released in the second and fourth quarters of the calendar year, and usually align with Microsoft Build and Microsoft Ignite.
```

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+| 3.3.0          | August 2024      |                  | Minor        | LTS          | 3.4.0                  |
+| 3.4.0          | September 2024   |                  | Minor        | LTS          | 3.5.0                  |
+| 3.5.0          | October 2024     |                  | Minor        | LTS          | _**6.0.0**_            |
+| 4.0.0          | November 2024    | ![Yes][1]        | Major        | STS          | 4.1.0                  |
+| 4.1.0          | December 2024    |                  | Minor        | STS          | 4.2.0                  |
+| 4.2.0          | January 2025     |                  | Minor        | STS          | 4.3.0                  |
+| 4.3.0          | February 2025    |                  | Minor        | STS          | 4.4.0                  |
+| 4.4.0          | March 2025       |                  | Minor        | STS          | 4.5.0                  |
+| 4.5.0          | April 2025       |                  | Minor        | STS          | _**7.0.0**_            |
+| 5.0.0          | May 2025         | ![Yes][1]        | Major        | LTS          | 5.1.0                  |
```
Comment (dcaro) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:121]:
```suggestion
| -------------- | ---------------- | ---------------- | ------------ | ------------ | ----------------------- |
| 2.61.0          | May 2024         | ![Yes][1]        | Major        | LTS          | 2.62.0                  |
| 2.62.0          | June 2024        |                  | Minor        | LTS          | 2.63.0                  |
| 2.63.0          | July 2024        |                  | Minor        | LTS          | 2.64.0                  |
| 2.64.0          | August 2024      |                  | Minor        | LTS          | 2.65.0                  |
| 2.65.0          | September 2024   |                  | Minor        | LTS          | 2.66.0                  |
| 2.66.0          | October 2024     |                  | Minor        | LTS          | _**2.79.0**_            |
| 2.67.0          | November 2024    | ![Yes][1]        | Major        | STS          | 2.68.0                  |
| 2.68.0          | December 2024    |                  | Minor        | STS          | 2.69.0                  |
| 2.69.0          | January 2025     |                  | Minor        | STS          | 2.70.0                  |
| 2.70.0          | February 2025    |                  | Minor        | STS          | 2.71.0                  |
| 2.71.0          | March 2025       |                  | Minor        | STS          | 2.72.0                  |
| 2.72.0          | April 2025       |                  | Minor        | STS          | _**2.90.0**_            |
| 2.72.0          | May 2025         | ![Yes][1]        | Major        | LTS          | 2.73.0                  |
```

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+
+**Examples:**
+
+* To qualify for support, you must update to 3.1.0 when released if you have Azure CLI 3.0.0 installed.
+* If Azure CLI 3.5.0 is the final release of Azure CLI version 3, it's supported until the release of Azure CLI 6.0.0.
+* When Azure CLI 5.0.0 is released, Azure CLI version 4 is no longer supported.
```
Comment (dcaro) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:127]:
```suggestion
* To qualify for support, you must update to 2.63.0 when released if you have Azure CLI 2.62.0 installed.
* If Azure CLI 2.66.0 is the last release of Azure CLI before breaking change in the fourth quarter of the calendar year, it's supported until the release of Azure CLI 2.79.0.
```

**Review by dcaro — CHANGES_REQUESTED** [review]
Thanks for the PR, please have a look at the comments.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md (hunk 2 above)
Comment (mikefrobbins) [reply docs-ref-conceptual/azure-cli-support-lifecycle.md:135]: Consider combining the two.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+  "azure-cli-support": "LTS-20240521",  
+  "azure-cli-telemetry": "1.0.8",  
+  "extensions": {  
+  }  
+}  
+```
+
+The following image illustrates the release cadence and support lifecycle of Azure PowerShell for LTS and STS releases.
+
+![Support Lifecycle](./media/support-lifecycle/support-lifecycle.png)
```
Comment (mikefrobbins) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:99]: This image is of Azure PowerShell.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+
+The following table provides examples of how the release cadence correlates with the support
+lifecycle:
+
+> [!NOTE]
+> The dates in the following table are for example purposes only. The actual release dates vary.
+
+| Version number | Release schedule | Breaking changes | Release type | Support type | Supported until version |
+| -------------- | ---------------- | ---------------- | ------------ | ------------ | ----------------------- |
+| 3.0.0          | May 2024         | ![Yes][1]        | Major        | LTS          | 3.1.0                  |
```
Comment (mikefrobbins) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:109]: The image in the _Breaking changes_ column is broken. You'll need the [checkmark](https://github.com/MicrosoftDocs/azure-docs-powershell/blob/main/media/shared/check-mark-button_2705.svg) image.

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 10]
```diff
@@ -0,0 +1,193 @@
...
+  "azure-cli": "2.48.0",  
+  "azure-cli-core": "2.48.0",  
+  "azure-cli-support": "LTS-20240521",  
+  "azure-cli-telemetry": "1.0.8",  
+  "extensions": {  
+  }  
+}  
+```
+
+The following image illustrates the release cadence and support lifecycle of Azure PowerShell for LTS and STS releases.
```
Comment (mikefrobbins) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:97]:
```suggestion
The following image illustrates the release cadence and support lifecycle of Azure CLI for LTS and STS releases.
```

File: docs-ref-conceptual/azure-cli-support-lifecycle.md [hunk 11]
```diff
@@ -0,0 +1,193 @@
...
+|3.9|October 2025
+|3.10|October 2026
+|3.11|October 2027
+
+### Operating systems
+
+Azure CLI can only be supported on operating systems where the above versions of Python are supported.
+
+- **Windows:** The currently supported versions of Windows client and server meet Python version requirements.
+- **macOS:** The currently supported version of maxOS 10.9 and above meet Python version requirements.
```
Comment (mikefrobbins) [inline docs-ref-conceptual/azure-cli-support-lifecycle.md:168]:
```suggestion
- **macOS:** The currently supported version of macOS 10.9 and above meet Python version requirements.
```

**Review by mikefrobbins — CHANGES_REQUESTED** [review]
See my comments

**Pull request closed without merging**